development.ini.example

```diff
@@ -123,7 +123,7 @@ fedora_epel_testing_master_repomd = https://download.fedoraproject.org/pub/epel/
 
 ## The base url of this application
 ## Used as the <base/> tag in the master template.
-base_address = http://0.0.0.0:6543/
+base_address = http://localhost:6543/
```

Comment:
@puiterwijk On a local box, 0.0.0.0 doesn't work for lots of the AJAX calls. Since developing on a local system is the common case and since this is the .example file, I think we really do want to change this to "localhost". Keep in mind that the readme directs developers to use Vagrant, so this example will not work for an out-of-the-box setup unless we accept this change.

Developers who want to work against a remote system are free to change their development.ini to be however they please, as the development.ini is in the gitignore. This file is just here to get you started.